test/lock-parser/fixtures/restriction-overwrite/out.json

```diff
@@ -0,0 +1,232 @@
+{
+  "dependencies": [
+    {
+      "name": "FSharp.Core",
+      "options": {
+        "restriction": "|| (== net45) (== portable-net45+win8+wp8+wpa81)"
```

Comment:
as per chat in person please let's not do any kind of resolution logic at this first step. grab all info as is do not apply any rules based on restrictions just yet. The more exact state we have to start with the easier it will be in the future to change logic, make other decisions based on this. And easier to debug

Reply:
Definitely makes sense from that perspective. Agree we should not be overwriting this early.